### PR TITLE
Generated by robot

### PR DESCRIPTION
**Pull request by caesarxuchao**
All 5 PRs generated in the same run of the robot:
https://github.com/kubernetes/client-go/pull/140
https://github.com/kubernetes/apimachinery/pull/10
https://github.com/kubernetes/apiserver/pull/4
https://github.com/kubernetes/kube-aggregator/pull/4
https://github.com/kubernetes/sample-apiserver/pull/2

### CODE REVIEW

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -389,6 +389,958 @@
 		{
 			"ImportPath": "gopkg.in/yaml.v2",
 			"Rev": "53feefa2559fb8dfa8d81baad31be332c97d6c77"
+		},
+		{
+			"ImportPath": "k8s.io/apimachinery/pkg/api/equality",
```
Comment (sttts) [inline Godeps/Godeps.json]: The kube-aggregator is not a library, but a binary. So vendoring `k8s.io/*` including the files is ok IMO. @deads2k wdyt?

Comment (deads2k) [inline Godeps/Godeps.json]: > The kube-aggregator is not a library, but a binary. So vendoring k8s.io/* including the files is ok IMO. @deads2k wdyt?

I think this is only updating the godeps.json manifest and not bringing in the vendored files.  I think the approach here is reasonable.

**Comment by deads2k** [comment]
@caesarxuchao the godep save step is going to add the k8s.io/* godep entries back into godeps.json. That's why the sync script in this repo checked out an older version of godeps.json every time it ran.

**Comment by luxas** [comment]
@deads2k In the future, can we have the vendor folder automatically synced to all these repos?
That will make the repos so much easier to consume, no vendor hacking would be needed then...

**Comment by deads2k** [comment]
> @deads2k In the future, can we have the vendor folder automatically synced to all these repos?
> That will make the repos so much easier to consume, no vendor hacking would be needed then...

All the non-k8s.io packages are vendored already.  Including the k8s.io packages makes a mess if you want to compose repo1 with repo2, so we aren't doing that.  Instead, the infrastructure should make the sync across the different k8s.io repos consistent.

**Comment by deads2k** [comment]
lgtm

**Comment by deads2k** [comment]
merging in preparation for turning on the auto-sync scripts

**Pull request merged**